internal/pkg/service/cli/cmd/remote/create/table/create_table_test.go

```diff
@@ -2,6 +2,7 @@ package table
 
 import (
 	"fmt"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
```

Comment:
We don't want such unrelated dependency.

Please add internal method `ptr`, or create an helper pkg, e.g, `service/common/ptr`

```
func ptr[T any](v T) *T {
	return &v
}
```